ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/preload-image/provision/cucushift-installer-rehearse-nutanix-ipi-preload-image-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-nutanix-ipi-preload-image-provision
+  steps:
+  - chain: ipi-conf-nutanix
+  - ref: ipi-conf-nutanix-preload-image
+  - chain: ipi-install
+  - chain: ipi-install-nutanix
+  - ref: enable-qe-catalogsource
+  - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-nutanix-vip
+  - ref: cucushift-installer-check-nutanix-infra
```

Comment:
Can we develop a step to check the installation is using pre-loaded image to run the install ?

Reply:
I'm trying to add a step to check installation is using pre-loaded image, when testing manually, I need to check following
"Using the existing rhcos image \\\"$image_name\\\" in PC"
exist in .openshift_install.log
In order to get the log file, is it OK to add following code [here]? Or is there other methods? (https://github.com/openshift/release/blob/cdaea73b987aa30e5ae8f6dc7a12fdbe5fcf2cfe/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L239)

```
  if [[ "${CLUSTER_TYPE}" == "nutanix" ]]; then
      cp "${dir}/.openshift_install.log" "${SHARED_DIR}/.openshift_install.log"
  fi
```